README.md

```diff
@@ -678,7 +678,8 @@ The encrypted binary will have the following structure:
 - Padding to ensure the encrypted length is a multiple of 4 words
 - Signature metadata block
 
-The AES key must be provided as a .bin file of the 256 bit AES key to be used for encryption.
+The AES key must be provided as a .bin file containing a 4-way share of the 256 bit AES key to be used for encryption.
+Eg if the 256 bit key is X, the bin file should contain the 256 bit numbers A B C D where X = A ^ B ^ C ^ D.
```

Comment:
This is the line that confused me (it implies that you know X, and then use that to generate A B C D). Perhaps it should be reworded slightly?
"E.g. if the bin file contains the 256-bit numbers A B C D, the 256-bit key X is calculated as X = A ^ B ^ C ^ D." :man_shrugging:  (Although my terminology may be wrong, as this is the first time I've encountered "key shares")

Reply:
You can do it either way round - if you want a specific key X then you can generate A, B, C & D to match your X, although have to be very careful to make sure they are properly random. The normal use case would be you don't need a specific key, so just randomly generate A, B, C & D and use whatever X that gives, so rewording is probably sensible